### ISSUE TITLE
Mob Spawner Not Working - Unable to insert souls.

### ISSUE THREAD

**Issue by beanzen** [brandon3055/Draconic-Evolution#848]
## Description
I'm using Draconic Evolution 1.10.2-2.1.1.236 with Minecraft 1.10.2 and Forge 2297. The mod pack is technically custom, but it's just FTB with a few things stripped out of it. 

I can craft "stabilized spawners" in the crafting grid with Ender IO broken spawners (specifically broken Blaze spawners collected from the nether) and a draconic core (any tier). When placed in the world, they have no soul in them, which I believe is expected behavior. I am trying to set up a blaze spawner in the overworld, and I am unable to insert a blaze soul into the stabilized spawner. The soul is consumed when I right click with the soul on the spawner, but the spawner remains empty and nothing happens.

I have tried with basic, wyvern, and draconic tier stabilized spawners, and I have also tried in both the overworld and the nether, and I am unable to put a blaze soul into the stabilized spawner and nothing happens. I've also tried messing with sending it a redstone signal (as a last ditch effort) and that also seems to have no effect. 

## Base information
* Minecraft version: 1.10.2
* Minecraft Forge version: 12.18.3.2297
* Mod Pack: Custom (FTB with a few things removed, like actually additions)

## Crash report
```
CRASH REPORT HERE
```

**Comment by brandon3055** [comment]
Hmm... When you craft a spawner using the ender IO recipe it should retain whatever type the ender IO spawner was. The fact that its consuming souls suggests that it is at least trying to set the type but ether failing for some reason or maby its just not rendering.

If Is this single player or on a server? If you can please try reproducing this in a creative world. You can set the spawner type with a spawn egg. Also apply a draconic tier core so it will ignore spawn requirements. That should confirm if its a visual bug.

**Comment by beanzen** [comment]
Okay, I did some more testing.

There is a visual glitch that doesn't seem to show the mob inside the spawner, but that's not a big deal...

What I have found is that there may be a conflict with the "Torchmaster" mod (adds a "Mega Torch" which stops mobs spawning within a predefined/configured radius, but works fine with allowing vanilla and enderio spawning machines to function).

I am able to get the stabilized spawner working with hostile mobs when outside the range of a "mega torch", and passive mobs like sheep always seem to work (even within range of a mega torch), but hostile mobs like the blaze won't spawn no matter what I do, and just get extinguished if within the range of the mega torch. I don't know if this is an issue with DE or with the torchmaster mod, but there seems to be an incompatibility there.
